static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-ab-thrasher.js

```diff
@@ -0,0 +1,99 @@
+define([
+    'common/utils/config',
+    'common/utils/detect'
+], function (
+    config,
+    detect
+) {
+    return function () {
+        var self = this;
+
+        this.id = 'MembershipBundlesThrasher';
+        this.start = '2017-01-23';
+        this.expiry = '2017-03-02'; // Thursday 2nd March
+        this.author = 'Justin Pinner';
+        this.description = 'Test appetite for membership bundles';
+        this.showForSensitive = true;
+        this.audience = 0.05;  // 5% ~ 7,000 unique browser thrasher views per variant per day
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = 'People on UK network front with at least an 1140px wide display.';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'One landing page will accrue a higher number of intention interactions than the other';
+
+        this.canRun = function () {
```

Comment:
Just checked with Charlotte - we want to exclude existing paying members from seeing our test ("otherwise it gets weird!") so you probably want to add `canReasonablyAskForMoney` here, like:

https://github.com/guardian/frontend/blob/43905b6b4b/static/src/javascripts-legacy/projects/common/modules/experiments/tests/membership-engagement-banner-tests.js#L40